Package.swift

```diff
@@ -5,7 +5,7 @@ import PackageDescription
 let package = Package(
     name: "TabBar",
     platforms: [
-        .iOS(.v13)
+		.iOS("14")
```

Comment:
It is better to use `.v14` value here

Reply:
![image](https://github.com/user-attachments/assets/f9949b3c-c7d4-40e2-b5ae-5de7edd32669)

Am I missing a point?